.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## RVM Version
```

Comment:
I don't mind env being first or last. I was rather about simplifying it into one section instead of 2 like before (os and rvm separately)

I do mind though having steps, expected, actual - in that order. Thats how I have been taught to write bugs and that's how its more natural to reproduce them I think :)

We might add Description section on top to give people some space to write what they think about the issue...